code/api/src/modules/product/query.test.js

```diff
@@ -48,7 +137,62 @@ describe('product query', () => {
       }}`
       })
       .expect(200)
-    console.log(response.body)
```

Comment:
We don't need to display console.log in our tests because it's similar to exposing sensitive material. However, it's the closest we have to pry. Just remember to delete it after tests pass